docgen/src/conditional-display.md

```diff
@@ -6,15 +6,24 @@ nav_groups:
 nav_sort: 1
 ---
 
-Sometimes, you will need to render some widgets instead of another if the query hits no results or if there is currently
-no query used. 
+Sometimes, you will need to render some widgets or content instead of another. To do that you can use the `createConnector` 
+function we provide for [making your own widget](Customization.html#creating-your-own-connectors). 
 
-To do that you can use the `createConnector` function we provide for [making your own widget](Customization.html#creating-your-own-connectors). 
-
-Let's see some examples:
+Below are some typical use cases when using conditional display makes sense but don't hesitate to read the 
+[making your own widget documentation](Customization.html#creating-your-own-connectors), it will give you more details 
+about the api. 
 
 ## Conditionally displaying content based on query state
 
+The first example where conditional display may apply is: 
+
+**What if you want to display a particular content when the user has not enter a query yet?**
```

Comment:
You could just change the title to this ^ then remove the introduction sentences